services/api-server/setup.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 0.5.0
```

Comment:
Probably we should release a corresponding client. I will make a note to do that